api/src/integrationTest/resources/application-test.yml

```diff
@@ -105,5 +119,10 @@ csm:
         default-page-size: 5
     rbac:
       enabled: true
+    s3:
+      endpointUrl: "http://localhost:9000"
+      bucketName: "test-bucket"
+      accessKeyId: "s3_username"
+      secretAccessKey: "s3_password"
```

Comment:
This is not consistent with the `spring.cloud.aws.s3` content above. Should this be simply removed since the spring config should be sufficient to configure the S3 client?

Same for all the other test application files.

Reply:
In fact, we cannot removed these properties because in CsmPlatformProperties `s3` is **not** marked as nullable.
Removing this from `application-*-test.yaml` will lead to an error when Spring will try to build the applicationContext. In particular, when Spring will try to build CsmPlatformProperties bean because s3 is expected in bean's definition.